app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantController < ApplicationController
+
+  def show
+    service = RevItUpService.new(current_user.lat, current_user.lng, meters_to_miles, params["price"])
```

Comment:
Action to render the random restaurant based off the information from our sinatra, rev-it-up application.